pkg/codegen/main.go

```diff
@@ -53,11 +54,11 @@ func main() {
 		panic(err)
 	}
 
-	obj.ObjectMeta.Annotations = map[string]string{
+	obj.(*unstructured.Unstructured).SetAnnotations(map[string]string{
```

Comment:
Is ObjectMeta gone ? We need to guarantee this annotation still gets set.

Reply:
[This change](https://github.com/rancher/wrangler/commit/c5ae8c7a7f241b4d2e766e965ccc7b6f85515f62#diff-099f7d83f8d29c3f530aa27406ae07b149faf56d91c1967c17eee2052d08197aR193) changed `obj` from `CustomResourceDefinition` to `runtime.Object`, which doesn't have an `ObjectMeta` property. What this is doing is casting the object to an `Unstructured` which we can call `SetAnnotations` on. The end result is the same, the annotation is set on the object.